timesketch/frontend-ng/src/components/LeftPanel/SigmaRules.vue

```diff
@@ -14,16 +14,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <template>
-  <div v-if="sigmaRules.length">
+  <div>
     <div class="pa-4" flat :class="$vuetify.theme.dark ? 'dark-hover' : 'light-hover'">
-      <span style="cursor: pointer" @click="expanded = !expanded"
-        ><v-icon left>mdi-sigma-lower</v-icon> Sigma Rules (<small
-          ><strong>{{ sigmaRules.length }}</strong></small
-        >)</span
-      >
+      <span style="cursor: pointer" @click="expanded = !expanded">
+        <v-icon left>mdi-sigma-lower</v-icon> Sigma Rules ({{ ruleCount }})
+      </span>
+      <span style="float: right">
```

Comment:
I opt to only display this if the panel is expanded. Just do a check for `expanded`

Reply:
done